daemon/qrexec-client.c

```diff
@@ -365,9 +314,16 @@ int main(int argc, char **argv)
                     VCHAN_BUFFER_SIZE, VCHAN_BUFFER_SIZE);
             if (!data_vchan) {
                 LOG(ERROR, "Failed to start data vchan server");
-                exit(1);
+                rc = 126;
```

Comment:
Please don't abuse 126 code for generic errors. If no better idea, you can leave it at 1 for now (here and in all the other places below). But, besides something from `sysexits.h` idea from the other PR, maybe declare 125 as "service call error"? Should be unlikely to conflict with the actual service exit code, I hope.

Reply:
That's an interesting proposal, but due to exactly what that would mean, it can easily confuse various applications that (possibly also some shells...) that it was indeed killed by a signal, or if something takes just 7 bits (excluding the "killed" case) it will even take it as a success. Yes, such applications are buggy. But also, lets not try running into such bugs.

Reply:
That is fair.

I think using `sysexits.h` codes is the best thing to do, except maybe if systemd defines a better standard.  That said, it does leak a small amount of information to the calling program, most notably if there was an out-of-memory condition.  Is that a problem?

Reply:
> codes

A singular code. A single one for any type of failing to start the allowed existing service call.